app/shared/constants/exchangeAccounts.js

```diff
@@ -0,0 +1,7 @@
+const exchangeAccounts = [
```

Comment:
What do you think of this setup to store constants that are used in different parts of the app? I can move it somewhere else if you think it doesn't belong here.

Reply:
If it's not hard, it might make sense. Otherwise it'll end up being in a smart contract at some point anyways, so putting too much effort into it wouldn't be productive.